lib/cc/presenters/pull_requests_presenter.rb

```diff
@@ -40,7 +46,9 @@ def coverage_message
       end
 
       def success_message
-        if @new_count > 0 && @fixed_count > 0
+        if @approved_by
```

Comment:
probably good to add `.present?` here to avoid a weird situation if it's an empty string